dirsrvtests/tests/suites/indexes/regression_test.py

```diff
@@ -163,8 +164,28 @@ def fin(always=False):
     return (indexes, attr)
 
 
-#unstable or unstatus tests, skipped for now
-@pytest.mark.flaky(max_runs=2, min_passes=1)
+def chech_dbi(dbsh, attr_name, expected = True, lowercase=False):
```

Comment:
Probably a typo - `chech_dbi -> check_dbi`

And then in other places where it's used